docs/docs/noir/concepts/ops.md

```diff
@@ -27,6 +27,7 @@ sidebar_position: 3
 | -         |           Subtracts two private input types together           |            Types must be private input |
 | \*        |          Multiplies two private input types together           |            Types must be private input |
 | /         |            Divides two private input types together            |            Types must be private input |
+| \%        |              Modulo operation                                  |                  Types must be integer |
```

Comment:
(github isn't letting me add a suggestion)

Can you change this to say "Remainder operation"?

Since `-21 % 4 = -1` rather than `3` as we'd expect for modulo.

Reply:
Done